src/wp-includes/widgets.php

```diff
@@ -1310,7 +1318,9 @@ function retrieve_widgets( $theme_changed = false ) {
 	$sidebars_widgets['wp_inactive_widgets'] = array_merge( $lost_widgets, (array) $sidebars_widgets['wp_inactive_widgets'] );
 
 	if ( 'customize' !== $theme_changed ) {
-		wp_set_sidebars_widgets( $sidebars_widgets );
+		// The second parameter is false to avoid recursion: refreshing the global
+		// $sidebars_widgets entails calling retrieve_widgets()
```

Comment:
Full stops required and multi-line comment format:

```suggestion
		/*
		 * The second parameter is false to avoid recursion: refreshing the global
		 * $sidebars_widgets entails calling retrieve_widgets().
		 */
```